init.sh

```diff
@@ -1,37 +1,29 @@
 #!/bin/bash
-SCRIPT_DIR=$PWD
-curl -s https://steamcdn-a.akamaihd.net/client/installer/steamcmd_linux.tar.gz | tar -vxz
 
-# Update / install server
-./steamcmd.sh +login $STEAM_USERNAME $STEAM_PASSWORD $STEAM_GUARD_TOKEN $STEAM_CMD_ARGS +force_install_dir $GAME_INSTALL_DIR +@sSteamCmdForcePlatformBitness 64 +app_update $GAME_ID +quit
+cd "$GAME_INSTALL_DIR" || exit
 
-# Move the steamclient
-mkdir -p /home/steam/.steam/sdk64/
-cp -f linux64/steamclient.so /home/steam/.steam/sdk64/steamclient.so
+ulimit -n 2048
 
-# Optionlly install RocketMod
-export MODULES_DIR=$GAME_INSTALL_DIR/Modules
-mkdir -p $MODULES_DIR
-cd $MODULES_DIR
+export TERM=xterm
+
+if [ -d "./Unturned_Headless_Data" ]; then
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:`dirname $0`/Unturned_Headless_Data/Plugins/x86_64/
+else
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:`dirname $0`/Unturned_Headless/Plugins/x86_64/
+fi
 
 if [ ! -d "$MODULES_DIR/Rocket.Unturned" ]; then
     if [ "$SERVER_TYPE" == "rm4" ]; then
-        curl https://ci.rocketmod.net/job/Rocket.Unturned/lastSuccessfulBuild/artifact/Rocket.Unturned/bin/Release/Rocket.zip -o Rocket.zip
-        unzip Rocket.zip
-        mv Modules/* ./
-        rmdir Modules
-        rm -rf ./Scripts
-        rm README
-        rm Rocket.zip
-    elif [ "$SERVER_TYPE" == "rm5" ]; then
-        curl -L https://ci.appveyor.com/api/buildjobs/bjt7acowdq73nh4u/artifacts/Rocket.Unturned-5.0.0.237.zip -o Rocket.zip
-        unzip Rocket.zip
-        rm README.md
-        rm LICENSE
-        rm Rocket.zip
-    fi    
+      mkdir /tmp/rm && \
+      cd /tmp/rm && \
+      curl -L https://github.com/RocketMod/Rocket.Unturned/releases/download/4.9.3.0/Rocket.Unturned.zip -o Rocket.zip && \
+      unzip Rocket.zip && \
+      mv Modules/* "$GAME_INSTALL_DIR"/Modules/ && \
+      mv Scripts/Linux "$GAME_INSTALL_DIR"/Scripts && \
+      cp -rf "$GAME_INSTALL_DIR"/Extras/Rocket.Unturned/ "$GAME_INSTALL_DIR"/Modules/ && \
+      rm Rocket.zip
+      rm . -rf
```

Comment:
this is no longer neccessary as rocketmod (now called LegallyDistinctMissile) is shipped with unturned, it just needs to copied into the modules folder; no need for download

Reply:
yes

Reply:
Fixed

Reply:
awesome